modules/Administration/Diagnostic.tpl

```diff
@@ -37,93 +37,128 @@
  * reasonably feasible for technical reasons, the Appropriate Legal Notices must
  * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  */
+*}
 
-/**
+<form name="Diagnostic" method="POST" action="index.php">
+    <h4>Quick Report</h4><br/>
```

Comment:
@pgorod
Could you pls add corresponding label for 'Quick Report' ?

Reply:
Done, thanks